rest-api/README.md

```diff
@@ -20,11 +20,17 @@ Make sure that you have google
 [cloud SDK](https://cloud.google.com/sdk/downloads) installed.
 
 From the rest-api directory, run:
-
+#### For Ubuntu
 * `sudo apt-get install mysql-server libmysqlclient-dev` (to install MySQL server and client —
 if you prefer to use docker, see note below)
+#### For Mac OS
+* `brew isntall mysql-connector-c`
+* If mysql is installed via homebrew you may have to symlink the mysql client:
```

Comment:
Do you want to describe how to install mysql via homebrew? Is the above command all you have to do?

Reply:
done